daemon/api.go

```diff
@@ -256,6 +256,7 @@ func sysInfo(c *Command, r *http.Request, user *auth.UserState) Response {
 		"os-release":     release.ReleaseInfo,
 		"on-classic":     release.OnClassic,
 		"managed":        len(users) > 0,
+		"forced-devmode": release.ReleaseInfo.ForceDevMode(),
```

Comment:
This is a mechanism to get us going but that we want to kill over time. It's also pretty bogus, because lack of apparmor does not mean lack of entire confinement, and definitely doesn't mean devmode. So we shouldn't be making it official by exposing it in a public API.

Here is a suggested alternative: let's have a command named:
```
$ snap debug confinement
```
the output should be one of: _strict, partial, or none_

For the implementation, let's call release.ReleaseInfo.ForceDevMode directly in the client command, instead of exposing it via the API. That buys us time to polish this without promising a public API.